bootcamp/models.py

```diff
@@ -20,43 +20,52 @@ def load(cls):
         return obj
 
 
+class PhoneNumber(models.Model):
+    type = models.CharField(max_length=300, null=False, blank=False)
+    phone = models.IntegerField(null=False, blank=False)
```

Comment:
better to set the phone as a string field.

Reply:
@shreeramneupane Done